README.md

```diff
@@ -40,12 +40,20 @@ An example for a "porcelain" API would be [ROS2](https://www.ros.org/). Others a
 | [RTA-VRTE](https://www.etas.com/en/products/rta-vrte.php) | [Adaptive AUTOSAR](https://www.autosar.org/standards/adaptive-platform/) platform software framework for vehicle computer from [ETAS GmbH](https://www.etas.com) |
 | [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) | Performant and robust open-source DDS implementation maintained by [ADLINK Technology Inc.](https://www.adlinktech.com/) |
 
-### Supported Platforms
+### Targeted quality levels & platforms
 
- * Linux
- * QNX
- * macOS (initial support - please test and report bugs)
- * Windows 10 (not yet - currently in progress)
+> [Quality level](./CONTRIBUTING.md#quality-levels) are 5..1, where 1 is highest
```

Comment:
Shall this table represent the target or the current state. I guess the first one. Do we also want to document the current state?

Reply:
we could also define iceoryx 1.0.0 should fulfil the requirements of the table, but we still should track the current state somewhere. Maybe like 42% done.

Reply:
Where would the 42% come from? Some points on this list are hard to measure. I'd suggest to provide coverage reports as described in #217

Reply:
well, my gut feelings are telling me it's the answer to the question how much we achieved 😀

Reply:
Can we measure your gut feeling with a sensor and upload it? :-D

Reply:
sure, but we need LoRa for that 😉